zuul.d/tempest_multinode.yaml

```diff
@@ -1,18 +1,20 @@
 ---
 - job:
-    name: cifmw-base-multinode-tempest
+    name: cifmw-base-multinode-master-tempest
     parent: cifmw-podified-multinode-edpm-base-crc
     timeout: 5400
-    abstract: true
     nodeset: centos-9-medium-crc-extracted-2-30-0-3xl
+    description: |
+      Base job definition for running test-operator with master release.
     vars:
       zuul_log_collection: true
       cifmw_deploy_edpm: false
-      openstack_release: antelope
+      openstack_release: master
```

Comment:
didn't they switch to `main` over `master` ?

Reply:
ohh yes, good catch

Reply:
https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-framework%20openstack_release&type=code